doc/table_of_contents_KR.md

```diff
@@ -0,0 +1,35 @@
+# 문서 목록
+
+## Grin 에 대한 설명들
+
+- [intro](intro.md) - Grin 에 대한 기술적인 소개
```

Comment:
Why not linking directly to `intro_KR.md`?

Reply:
Oh. My miss. 
Thank you :)